content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,374 @@
+  full URL needed to access the document through the event website.
+
+  An image can also be added to the wall by clicking the :guilabel:`🖉 (pencil)` icon that appears
+  when the cursor hovers over the :guilabel:`camera` box in the upper-right corner of the pop-up
+  window. Then, from the resulting file explorer window, select the desired image to be added to
+  the wall.
+
+  This field **only** appears on the event form if the *Social Marketing* app is installed, and an
+  *X* account has been added as a stream on the application.
+- :guilabel:`Template`: choose a pre-configured event template from the resulting drop-down menu.
+
+   Or, create a new one directly from this field, by typing in the name of the new template, and
+   selecting either: :guilabel:`Create` (which creates the template, and can be edited later) or
+   :guilabel:`Create and edit...` (which creates the template, and reveals a separate template page
+   to configure the template in greater detail).
+- :guilabel:`Tags`: add any corresponding tags to briefly describe the event (e.g. `Online`,
+  `Conference`, etc.). Multiple tags can be added per event.
+- :guilabel:`Organizer`: designate the organizer of the event.
+- :guilabel:`Responsible`: designate a user in the database to be responsible for this event.
+- :guilabel:`Company`: designate which company in the database to which this event is related. This
+  field **only** appears if working in a multi-company environment. If left blank, the event will
+  relate to all companies in the database.
+- :guilabel:`Website`: choose to restrict the publishing of this event to a specific website. If
+  this field is left blank, the event can be published on *all* websites in the database.
+- :guilabel:`Venue`: enter the pertinent event venue details in this field.
+- :guilabel:`Exhibition Map`: if desired, click the :guilabel:`Upload your file` button to Upload
+  a map of the event venue.
+- :guilabel:`Limit Registrations`: if this checkbox is ticked, a limit to the amount of
+  registrations is added to the event, and that desired limit amount **must** be entered in the
+  blank field before :guilabel:`Attendees`.
+- :guilabel:`Badge Dimension`: select a desired dimension for event badges. The options are:
+  :guilabel:`A4 foldable`, :guilabel:`A6`, or :guilabel:`4 per sheet`.
+- :guilabel:`Badge Background`: if desired, click the :guilabel:`Upload your file` button to upload
+  a custom background for event badges.
+
+When the fields in the event form have been adequately filled in, move on to the three tabs at the
+bottom of the event form for further customization.
+
+Those tabs are: :ref:`Tickets <create_events/event_form/tickets>`, :ref:`Communication
+<create_events/event_form/communication>`, :ref:`Questions <create_events/event_form/questions>`,
+and :ref:`Notes <create_events/event_form/notes>`.
+
+.. _create_events/event_form/tickets:
+
+Tickets tab
+-----------
```

Comment:
I like the way you formatted this; nice! This is easy to follow, with the tabs separated out